azure-pipelines.yml

```
@@ -0,0 +1,48 @@
+stages:
+- stage: build
+  displayName: Build Image
+  jobs:
+  - job: build_image
+    displayName: Build Image
+    pool:
+      name: "DAS - Continuous Integration"
+    steps:
+      - task: gittools.gitversion.gitversion-task.GitVersion@5
+        displayName: GitVersion
+        inputs:
+          configFilePath: GitVersion.yml
+          updateAssemblyInfo: true
+
+      - task: Docker@2
+        displayName: Login to Docker Hub
+        inputs:
+          command: login
+          containerRegistry: "Apprenticeships DevOps Docker Hub"
+
+      - task: Docker@2
+        displayName: Build Image
+        inputs:
+          command: build
+          dockerfile: Dockerfile
+          repository: apprenticeshipsdevops/das-self-service
+          tags: |
+            $(build.buildnumber)
```

So it aligns with below
```suggestion
          tags: $(build.buildnumber)
```